src/makeCalculator.js

```diff
@@ -5,6 +5,49 @@
  */
 function makeCalculator() {
```

Comment:
The function name `makeCalculator` is appropriate as it clearly describes the purpose of the function.